admin/release-appstarter

```diff
@@ -12,7 +12,7 @@ git checkout $branch
 echo -e "${BOLD}Build the framework distributable${NORMAL}"
 
 echo -e "${BOLD}Copy the main files/folders...${NORMAL}"
-releasable='app public writable README.md contributing.md env license.txt spark tests/_support'
+releasable='app public writable README.md contributing.md env license.txt spark tests/_support .gitignore'
```

Comment:
tests_support should actually not be in this list (my mistake). If you could remove it, that would save a change later :)

Reply:
Sure thing, all set!